_posts/blog/2018-10-18-review-living-archive.md

```diff
@@ -0,0 +1,20 @@
+
+The Living Archive of Aboriginal Languages was created in 2012 by the Charles Darwin University, the Australian National University and the NT Department of Education and Children’s Services, with funding from the Australian Research Council through the Linkage Infrastructure Equipment and Facilities scheme. On a second phase, the archive received financial aid again from the Research Council, and added three more educational institutions to the working team. All these organizations partnered up to gather the scattered books of twenty school-based Literature Production Centres at the Northern Territory, that from 1973 to early 2000 produced an amazing amount of indigenous knowledge recorded in very simple literary projects. 
+
+The “centres” were part of a discontinued Bilingual Program of northern Australia that aimed for the literacy of aboriginal children. Indigenous and non-indigenous people worked together to write and illustrate the books that contain “teaching materials, literacy primers, children’s stories, stories of local cultural significance such as environmental knowledge, traditional practices, oral literature, ethno-botany, history, non-sacred versions of Dreaming or Creation stories, experience stories, instructional manuals, cautionary tales, and many others” (Bow et al 348). The source of these documents were the oral narratives, and common knowledges of the indigenous communities in their own languages. Due to changes to educational policies, by 2010 the majority of the Centres closed, and the schools managed the materials in different ways that range from using them for educational purposes to actually destroying them (Bow et al 347-250). Today, some of the physical books are available in special collections of university libraries, and the rest, a large number, are dispersed within the communities and local educational centers and institutions. Therefore, access to these books and the indigenous knowledges they hold was selective and/or difficult before the creation of the digital collection.
+
+Access is an guiding force for this archive, but for whom is the key question. Jeannette A. Bastian states “the creation of collective memory by nations, communities, or groups of people depends upon their ability to access their own cultural heritage” (81). For Bastian, access should be a fundamental component of custody. The Living Archive takes this insightful argument in practice as an example of postcustodial archiving. The project does not include keeping the original material they tracked down, rather is invested in making the books reachable for a diverse group of people –linguists, teachers, students, researches, as their website guide shows. But more important is its redress to the communities that created the books, because the digital collection liberated their own cultural heritage from depending on the aftermath power infrastructures and custody of a discontinued bilingual educational program. 
```

Comment:
Sentence 1: "is **a** guiding force"
Sentence 5 (the project does not include...): this is a bit confusing. Maybe just "the project does not keep.... instead, it is invested"?
Final sentence: this is also a bit confusing, perhaps because it's hard to identify the subject of the verb "depending". Is it the cultural heritage, or the communities? It might be simpler as two sentences.